docs/recipes.rst

```diff
@@ -148,3 +148,29 @@ Also don't forget to rename the folder of your local working copy as Rally will
     mv teamtrackrepo default
 
 From now on, Rally will treat your repository as default and you need to run Rally with ``--track-repository=old-rally-default`` if you want to use the out-of-the-box Rally tracks.
+
+
+.. _recipe_testing_rally_against_ccr_clusters:
+
+Testing Rally against CCR Clusters using a remote metric store
```

Comment:
Nit: clusters (lower case)

Reply:
Fixed in https://github.com/elastic/rally/pull/676/commits/7202a0019a53e0220015b8928ef981d823d38241